### ISSUE TITLE
SLD database not (obviously) linked to from web site

### ISSUE THREAD

**Issue by paulmillar** [reflectivity/edu_outreach#43]
Issue #2 was closed with the creation of a service.  However, that URL is not obviously linked to from the https://www.reflectometry.org/ website.

Perhaps it would be useful to include a link from that site to the SLD database, or to make an existing link more prominent.

**Comment by arm61** [comment]
As I understand, though @wpotrzebowski or @aglavic can probably explain better, the SLD database is currently offline for maintenance.

**Comment by wpotrzebowski** [comment]
Yes, there's been a python update on the server pushed by the administrator (to support the latest functionality of SLDDB) and the service went offline. I will try to bring it back on-line.

**Comment by tomarnoldess** [comment]
We are restructuring the website to make links to important orso activities more obvious. when were done there will be an obvious link from the front page of the website.
